### ISSUE TITLE
Create initial game calculations module

### ISSUE THREAD

**Issue by Muthaias** [Greenheart/swipeforfuture.com#54]
-   Support complex calculations and value dependencies
-   Values should have dependencies
-   Values can be fixed or dynamic

    -   Fixed values only updates from card logic
    -   Dynamic values uses its depencencies to calculate a value

-   Values can be exposed or hidden
    -   Hidden values are only used for logic and calculations
    -   Exposed values should have an icon and will be visible in the UI

## Implementation

-   Generic or specific implementation?
-   Atleast we might want to consider creating a generic Interface for the `Game`

```
interface GameCalculations {
    exposedValues: {
        id: string
        icon: string
        value: number
    }[]
    values: {
        id: string
        value: number
    }
    flags: {
        id: string
        value: boolean
    }
    getValue(id: string): number
    setValue(id: string, value: number): void

    getFlag(id: string): boolean
    setFlag(id: string, boolean): void
}
```

**Comment by Greenheart** [comment]
Is this still relevant? I feel like this could be considered resolved when #156 and #157 have been merged

**Comment by Greenheart** [comment]
Closing as legacy. Could potentially be implemented with Tick Modifiers
